recipe/meta.yaml

```
@@ -7,44 +7,71 @@ package:
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: d48c2794835303388504757e11a47c2471616e1417b8a8b6f5fe0452595da5bc
+  url: https://github.com/winedarksea/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 6811694c415abbeca8eba13c017cfc366d3b060f4873cf39cbaa4ea8a358ac57
 
 build:
   number: 0
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install . --no-deps -vv --no-build-isolation
+  # skipping s390x due to test dependencies not being on platform
+  skip: true  # [py<37 or s390x]
 
 requirements:
   host:
     - pip
-    - python >=3.7
+    - python
+    - setuptools
+    - wheel
   run:
-    - python >=3.6
+    - python
     - numpy >=1.14.6
-    - pandas >=0.25.*
+    - pandas >=0.25.0
     - scikit-learn >=0.20.*
     - statsmodels >=0.10.*
-    - requests
+
```

I see some version pinnings in the `additional` extra. I think we should add them in the `run_constrained` section:

```suggestion
  run_constrained:
    - holidays >=0.9
    - prophet >=0.4.0
    - xgboost >=1.6
    
```

---

I'll get these added!